pgmpy/models/BayesianModel.py

```diff
@@ -373,11 +373,13 @@ def dfs(node):
             return descendents
 
         from pgmpy.independencies import Independencies
```

Comment:
@yashu-seth It has been imported already [line 13](https://github.com/pgmpy/pgmpy/pull/545/files#diff-5486bc43772bb0d18a7231f9496abcb7R13). Also IndependenceAssertion can also be imported in same statement.